windows-installer/Makefile

```diff
@@ -0,0 +1,27 @@
+NISNOPTS=""
+
+!IFDEF INSTVERSION
+NSISOPTS=$(NSISOPTS) /DVERSION=$(INSTVERSION)
+!ENDIF
+!IFDEF INSTBUILD32
+NSISOPTS=$(NSISOPTS) /DBUILD32=$(INSTBUILD32)
+!ENDIF
+!IFDEF INSTBUILD64
+NSISOPTS=$(NSISOPTS) /DBUILD64=$(INSTBUILD64)
+!ENDIF
+!IFDEF INSTLICENSE
+NSISOPTS=$(NSISOPTS) /DLICENSE_FILE=$(INSTLICENSE)
+!ENDIF
+
+openssl-installer: openssl.nsi
+	echo $(NSISOPTS)
+	makensis.exe $(NSISOPTS) /V4 .\openssl.nsi
+
+signed-openssl-installer: openssl.nsi
+	makecert.exe /n "CN=TestCompany" /r /h 0 /eku "1.3.6.1.5.5.7.3.3,1.3.6.1.4.1.311.10.3.13" /sv testcert.pvk testcert.cer

```

Comment:
is it intentional to use "1.3.6.1.5.5.7.3.3,1.3.6.1.4.1.311.10.3.13" as extended key usage here:
or is it here just for testing? If I understand this guide right:
https://learn.microsoft.com/en-us/windows/win32/appxpkg/how-to-create-a-package-signing-certificate
then we should be rather using "1.3.6.1.5.5.7.3.3". That's at least my very limited understanding of the matter.

Reply:
Its subtle, note the comma in the middle of the string.  Its actually two eku oids.  See step 2 in the link above.  I'm not sure if /eku can be specified twice with that utility.

Reply:
oh man, I need glasses. indeed there is a comma. all good thanks.